commit 4ae68c2fc2247b15f8f65b9ffec00257e7cdf4f8 (refs/changes/44/2544/3)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2017-09-15T01:50:57+00:00 (2 years, 1 month ago)
    
    OS-5858 Want support for X722
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Dan McDonald <danmcd@joyent.com>

diff --git a/overlay/generic/etc/driver_aliases b/overlay/generic/etc/driver_aliases
index 8959edc1..c3836113 100644
--- a/overlay/generic/etc/driver_aliases
+++ b/overlay/generic/etc/driver_aliases
@@ -499,6 +499,12 @@ i40e "pciex8086,1586"
 i40e "pciex8086,1589"
 i40e "pciex8086,158a"
 i40e "pciex8086,158b"
+i40e "pciex8086,37ce"
+i40e "pciex8086,37cf"
+i40e "pciex8086,37d0"
+i40e "pciex8086,37d1"
+i40e "pciex8086,37d2"
+i40e "pciex8086,37d3"
 i915 "pci8086,2562"
 i915 "pci8086,2572"
 i915 "pci8086,2582"
